src/contentmanager.cpp

```diff
@@ -31,7 +31,8 @@ void ContentManager::setLocal(bool local) {
         return;
     }
     m_local = local;
-    emit(remoteParamsChanged());
+    if (!local)
+        emit(remoteParamsChanged());
```

Comment:
There is several place where the signal `remoteParamsChanged` is emit.
It would be better to move the test of `m_local` in the listener of the signal (`updateRemoteLibrary`)